seed/migrations/0114_rehash_postal_code.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# This rehashing file should be used in the future if needed as it has been optimized. Rehashing takes
+# awhile and should be avoided if possible.
+from __future__ import unicode_literals
+
+# import time
+# from datetime import timedelta
+
+from django.db import connection, migrations, transaction
+
+from seed.data_importer.tasks import hash_state_object
+
+import re
+
+
+def forwards(apps, schema_editor):
+    ps = apps.get_model('seed', 'PropertyState')
+    ts = apps.get_model('seed', 'TaxLotState')
+
+    def zero_fill(postal):
+        if postal:
+            if bool(re.compile(r"(\b\d{1,5}-\d{1,4}\b)").match(postal)):
+                print(postal.split('-')[0].zfill(5) + '-' + postal.split('-')[1].zfill(4))
+                return postal.split('-')[0].zfill(5) + '-' + postal.split('-')[1].zfill(4)
+            elif bool(re.compile(r"(\b\d{1,5}\b)").match(postal)):
+                print(postal.zfill(5))
+                return postal.zfill(5)
+
+    for field in ps.objects.all():
+        field.postal_code = zero_fill(field.postal_code)
+        field.owner_postal_code = zero_fill(field.owner_postal_code)
+        field.save()
+
+    for field in ts.objects.all():
+        field.postal_code = zero_fill(field.postal_code)
+        field.save()
+
+    property_sql = (
+        "UPDATE seed_propertystate " +
+        "SET created = seed_propertyauditlog.created, updated = seed_propertyauditlog.created " +
```

Comment:
I think we'd only need to restore the `created` field and not the `updated` field here

Reply:
I actually think we don't need to SET the `created` nor `updated` field in either of these... just the `postal_code`, right?

Reply:
I updated my comment (meant to switch `created` and `updated`). I'd think we'd need to "revert" the `updated` field since changing postal_codes one at a time using `save()` will change the `updated` time for each record.

Reply:
I guess, I'm just assuming we wouldn't actually want the `updated` times to change, but I could definitely understand wanting those values to change.

Reply:
Ah, that makes sense. I think it is okay if the updated times change as it really is a change.